src/app/myPayments/tests/myPayments.spec.js

```diff
@@ -1,58 +1,51 @@
-    var scope,
-        q,
-        oc;
-    beforeEach(module('orderCloud'));
-    beforeEach(module('orderCloud.sdk'));
-    beforeEach(inject(function($q, $rootScope, OrderCloud) {
-        scope = $rootScope.$new();
-        q = $q;
-        oc = OrderCloud;
-    }));
-
     describe('State: myPayments', function() {
-        var state;
-        beforeEach(inject(function($state) {
-            state = $state.get('myPayments');
+        var myPaymentsState;
+        beforeEach(function() {
+            myPaymentsState = state.get('myPayments');
             spyOn(oc.Me, 'ListCreditCards').and.returnValue(null);
             spyOn(oc.Me, 'ListSpendingAccounts').and.returnValue(null);
-        }));
-        it('should resolve UserCreditCards', inject(function($injector){
-            $injector.invoke(state.resolve.UserCreditCards);
-            expect(oc.Me.ListCreditCards).toHaveBeenCalledWith(null, null, null, null, null,{'Editable': true});
-        }));
-        it('should resolve UserSpendingAccounts', inject(function($injector) {
-            $injector.invoke(state.resolve.UserSpendingAccounts);
-            expect(oc.Me.ListSpendingAccounts).toHaveBeenCalledWith(null, null, null, null, null, {'RedemptionCode': '!*'});
-        }));
-        it('should resolve UserSpendingAccounts', inject(function($injector) {
-            $injector.invoke(state.resolve.GiftCards);
-            expect(oc.Me.ListSpendingAccounts).toHaveBeenCalledWith(null, null, null, null, null, {'RedemptionCode': '*'});
-        }));
+        });
+        it('should resolve UserCreditCards', function(){
+            mock.Parameters = {
+                filters: {
+                    Editable: true
+                }
+            };
+            injector.invoke(myPaymentsState.resolve.UserCreditCards);
+            expect(oc.Me.ListCreditCards).toHaveBeenCalledWith(mock.Parameters);
+        });
+        it('should resolve UserSpendingAccounts', function() {
+            mock.Parameters.filters = {
+                RedemptionCode: '!*'
+            };
+            injector.invoke(myPaymentsState.resolve.UserSpendingAccounts);
+            expect(oc.Me.ListSpendingAccounts).toHaveBeenCalledWith(mock.Parameters);
+        });
+        it('should resolve UserSpendingAccounts', function() {
+            mock.Parameters.filters = {
+                RedemptionCode: '*'
+            };
+            injector.invoke(myPaymentsState.resolve.GiftCards);
+            expect(oc.Me.ListSpendingAccounts).toHaveBeenCalledWith(mock.Parameters);
+        });
     });
 
     describe('Controller: MyPaymentsController', function() {
         var myPaymentCtrl,
-            state,
-            toaster,
-            exceptionHandler,
-            confirm,
-            creditCardModal,
             authNet,
             mockCreditCard,
             mockSpendingAccount,
             mockGiftCard,
             mockCCresponse,
             mockSAresponse,
```

Comment:
A lot of these injections and mock payments (credit card, spending account, gift card) we should move to the global.spec.js that way we can: declutter unit tests AND if we need access to a mocked value we can reuse it.